SConstruct

```diff
@@ -180,6 +172,24 @@ opts.Add(
 
 opts.Add(BoolVariable("build_library", "Build the godot-cpp library.", True))
 
+# CPU architecture options.
+architecture_array = ["auto", "universal", "x86_32", "x86_64", "arm32", "arm64", "rv64", "ppc32", "ppc64", "wasm32"]
+architecture_aliases = {
+    "x64": "x86_64",
```

Comment:
In godot I also added `x86`.

Reply:
Done, added `x86` as an alias for `x86_32`.